go.sum

```diff
@@ -329,6 +331,7 @@ github.com/oklog/run v1.0.0/go.mod h1:dlhp/R75TPv97u0XWUtDeV/lRKWPKSdTuV0TZvrmrQ
 github.com/onsi/ginkgo v1.6.0/go.mod h1:lLunBs/Ym6LB5Z9jYTR76FiuTmxDTDusOGeTQH+WWjE=
 github.com/onsi/ginkgo v1.8.0/go.mod h1:lLunBs/Ym6LB5Z9jYTR76FiuTmxDTDusOGeTQH+WWjE=
 github.com/onsi/gomega v1.5.0/go.mod h1:ex+gbHU/CVuBBDIJjb2X0qEXbFg53c61hWP/1CpauHY=
+github.com/opentracing/opentracing-go v1.1.0/go.mod h1:UkNAQd3GIcIGf0SeVgPpRdFStlNbqXla1AfSYxPUl2o=
```

Comment:
See discussion here https://github.com/open-telemetry/opentelemetry-collector/pull/4423#discussion_r756295333

Reply:
@pavankrish123, are you able to use the v2 module with this PR? I was able to get the opentracing dependency removed when using the latest commit from the v2 branch, once https://github.com/grpc-ecosystem/go-grpc-middleware/pull/477 got merged.

Reply:
Thanks @jpkrohling - Unfortunately v2 doesn't support settable nor has apis for setting global grpc logger as a matter of fact for some reason. Only interceptor logs are available on the branch which will not produce framework level logs. One way to go about it is to replicate `Settable` code in our code base to remove opentracing dep (basically mutex protected getters and setters). Let me know WDYT

Reply:
https://github.com/grpc-ecosystem/go-grpc-middleware/issues/479

Reply:
I would certainly prefer to apply this change upstream and consume it here, but if they don't want to have this for v2, it's certainly OK to copy this code. Just make sure to retain the copyright notice for the files you copy.

Reply:
Thanks @Juraci Paixão Kröhling - I have worked around `SettableLoggerV2` and issued a new PR. The PR removes the dependency om grpc-middleware completely. Can you please take a look and see if it is ok. Same test - same result.